src/measurement/vmsa.rs

```diff
@@ -113,6 +113,71 @@ where
     }
 }
 
+impl<T, const N: usize> TryFrom<Vec<T>> for LargeArray<T, N>
```

Comment:
Do we want to implement this for Vec, or should we adjust this to do it to a slice?
```suggestion
impl<T, const N: usize> TryFrom<&[T]> for LargeArray<T, N>
```
We will still want to clone/copy the data, but it may save a move action.